### ISSUE TITLE
NHM 3.0.0.5  and .NET Framework 4.8

### ISSUE THREAD

**Issue by Miiichal** [nicehash/NiceHashMiner#1997]
Unable to run NHM 3.0.0.5, because it needs .NET Framework 4.8

But we have few servers with databases, which are working only with .Net 4.5 (some backup programs etc). Is there a way to run NHM 3.0.0.5 with older .NET?

**Comment by luc1an24** [comment]
Hi @Miiichal 
To use NHM 3.0.0.5 you will have to use .NET Framework 4.8 since some functionalities are incompatible with previous versions.

If you have any other questions please feel free to ask.
Thank you for understanding.

**Comment by goodwelling** [comment]
Thanks michal   About me II love to have works but not in the deals of
computers so I really need help
How could we change that frame and other things I need a partner 😊
---------- Forwarded message ---------

**Comment by GSUcoiners** [comment]
After many days I am Tired of 3005 already.  Benchmarks are harder to work with then 19222.  Keep getting NONE in the benchmark field.  Did not have these problems with Net 45.  Been trying to isolate problem.  My hardware is H110 and A970 all with Win7 and GPUs1080sto1060s.  Had to back rev to Net452 and19222 to get ALL benchmarks to work again.  Is this a win7 problem?

**Comment by o-Epictetus-o** [comment]
Windows 7 is no longer a supported operating system by microsoft

**Comment by Miiichal** [comment]
We are not discussing about supported systems. **Its only about .NET**
Lot of miners have their rigs on Win7 cause of better management od GPU memory.

**Comment by GSUcoiners** [comment]
I have been able to get a better hashrates from window 7 than windows 10.  Lot of overhead in windows 10.  And I have not needed windows support for years.  I use only a few hotfixes from microsoft.  LOL... Id use Windows 98 if it was more profitable.  But back to NET48, I contacted another miner that has experimented with some fixes.  We are trying to see if we can use 3005 with net452.  It may limit some functions but that is OK.  I will post if we have success.

**Comment by luc1an24** [comment]
Hi @Miiichal  and @GSUcoiners 
How does NET 4.5 works with you?
If you switched to 4.8 do you have any problems?
Have a nice day!

**Comment by rgaufman** [comment]
When I try to run NiceHash it says I need .NET 4.8 but when I try to install it, it says it is not supported in Windows 10. Any solution?

**Comment by GSUcoiners** [comment]
We are still using mostly win 7 but for amd cards to use the compute mode we are using win 10. So we downloaded from MS .net48 standalone installer.  we have an internet cap and so downloading and reusing keeps us from using too much bandwidth.  SO The .net48 standalone installer will install on both win 7 and win 10 with no problems.  Not sure if this workaround will work for you but we have no problems.  We also needed to SHA2 standalone installers for win 7 to install the newer nvidia drivers too. we just search the name of the MS KB or .net 48 and download.  This method helps us a lot as we get no interrupted downloads from ms during install (network install) and we can reuse the Standalone Installer over and over.  Hope this helps.  Stay Well!
